src/olympia/devhub/templates/devhub/addons/owner.html

```diff
@@ -8,11 +8,6 @@
 {{ dev_page_title(title, addon) }}
 {% endblock %}
 
-{% block bodyclass %}
-{{ super() }}
-{% if not check_addon_ownership(request, addon) %} no-edit{% endif %}
-{% endblock %}
```

Comment:
This is done in the view now, and thanks to the other change to `devhub/base.html`, the check is only done once.